src/main/java/sleuth/webmvc/Frontend.java

```diff
@@ -1,5 +1,7 @@
 package sleuth.webmvc;
 
+import org.jboss.logging.MDC;
```

Comment:
hopefuly this is the wrong MDC :D

Reply:
Oops, still works though. 😊 
Looks like it's brought in by Hibernate Validator.